swift/ql/src/queries/Security/CWE-757/TLS13IsNotUsed.ql

```diff
@@ -0,0 +1,58 @@
+/**
+ * @name TODO
+ * @description TODO
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity TODO
+ * @precision high
+ * @id swift/tls13-is-not-used
```

Comment:
Consider a slightly more generic `@id`, in case requirements are increased at some point.